### ISSUE TITLE
`<ui5-shellbar>` Axe Issues

### ISSUE THREAD

**Issue by codefactor** [SAP/ui5-webcomponents#3953]
### **Bug Description**
This is a break out of the various Shellbar issues documented in the following issue:
https://github.com/SAP/ui5-webcomponents/issues/3933

#### 1. Elements must only use allowed ARIA attributes

_Maybe `<ui5-buttons>` should not have an aria-label, maybe title is enough?_

This one points to several `<ui5-button>` tags in the `<ui5-shellbar>`

```html
<ui5-button icon="sap-icon://bell" data-ui5-text="Notifications" data-ui5-stable="notifications" id="ui5wc_47-item-2" style="order:3;" class=" ui5-shellbar-button ui5-shellbar-bell-button" data-ui5-notifications-count="99+" aria-label="99+ Notifications" title="99+ Notifications" ui5-button="" icon-only="" has-icon="">
```

It also says...

> Fix the following:
>ARIA attribute is not well supported on the element and the text content will be used instead: aria-label

It seems like Axe doesn't like this web-component to have an "aria-label" attribute... I guess?
There is also a title attribute, maybe the aria-label should be removed here?

Here is where the title/aria-label are added:
https://github.com/SAP/ui5-webcomponents/blob/f6dee74efda5bf7e59883f72bbe00aa4cb834b1e/packages/fiori/src/ShellBar.hbs#L111-L112

Maybe this should only be title?

This might be more wide-spread on multiple .hbs files where an `aria-label` and `title` are added to a `<ui5-button>` when only a `title` attribute would be enough.

### **Expected Behavior**
Shellbar should have no axe errors

### **Steps to Reproduce**
See https://github.com/SAP/ui5-webcomponents/issues/3933

### **Isolated Example**
N/A

### **Context**
- UI5 Web Components version: {...}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Bussiness impact: {...}

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-p could you check this issue related to the buttons inside the ShellBar, if they are implemented by spec. According to the author's words, his checks report issues when the button has both title and aria-label, could that be wrong, see the details above?

**Comment by ilhan007** [comment]
Hi @codefactor this is fixed in the master and will be available in 0.31 with the next patch.

**Comment by ilhan007** [comment]
Fixed in 0.31.19
